lib/src/dart2_suggestors/pubspec_over_react_upgrader.dart

```diff
@@ -25,9 +25,15 @@ import 'package:source_span/source_span.dart';
 class PubspecOverReactUpgrader implements Suggestor {
   /// Version constraint that ensures a version of over_react compatible with
   /// the new forwards- and backwards-compatible component boilerplate.
-  static final VersionRange overReactConstraint =
+  static final VersionRange dart1And2Constraint =
       VersionConstraint.parse('^1.30.2');
 
+  /// Version constraint that ensures a version of over_react compatible with
+  /// the Dart 2 builder and also opens the range up to over_react 2.x which is
+  /// the first release that drops support for Dart 1.
+  static final VersionRange dart2Constraint =
+      VersionConstraint.parse('>=1.30.2 <3.0.0');
```

Comment:
We'll need to raise this as we release additional versions that include the builder and any other fixes that may be necessary.